evm/db/journal.py

```diff
@@ -1,3 +1,5 @@
+from typing import Dict  # noqa: F401
```

Comment:
There should be no blank lines between stdlib imports, and they should be ordered alphabetically

Reply:
@carver implemented automated import sorting using [`isort`](https://github.com/timothycrosley/isort).  That's like a nice half-step towards what `black` does.

Reply:
Yup, I've been happy with `isort`. Here are the configs we put in `tox.ini`, which should be close enough to the py-evm standards to drop in place with only a few edits:

https://github.com/ethereum/web3.py/blob/307dc879f18bec9cc0071fc2a6205c1d54b985f5/tox.ini#L9-L19

```
[isort]
combine_as_imports=True
force_sort_within_sections=True
include_trailing_comma=True
known_standard_library=pytest
known_third_party=lru,eth_tester  # customize
known_first_party=web3,ens  # customize
line_length=21
multi_line_output=3
skip=web3/main.py,web3/utils/windows.py  # customize
use_parentheses=True
```

Reply:
Thanks for sharing @carver! I could give that a try. We also have a [tracking issue](https://github.com/ethereum/py-evm/issues/476) regarding black. I think moving to black would be my favorite option but it would be great to get more opinions on it :)